source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnRenderer.mm

```diff
@@ -6,48 +6,46 @@
 //
 
 #import "ACRColumnRenderer.h"
+#import "ACRColumnView.h"
 #import "ACRRegistration.h"
 #import "Column.h"
 #import "SharedAdaptiveCard.h"
 
 @implementation ACRColumnRenderer
 
-+ (ACRColumnRenderer* ) getInstance
```

Comment:
There's three different styles for this throughout the PR. Let's stick with one.

---
In reply to: [128559390](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128559390) [](ancestors = 128559390)